package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@mozilla-ai/mcpd",
+  "version": "0.1.0",
+  "description": "JavaScript/TypeScript SDK for interacting with mcpd",
```

Comment:
I think we need to add a `"type": "module"` to support es6 imports https://stackoverflow.com/questions/61401475/why-is-type-module-in-package-json-file